components/crowdfunding-redesign/ContentOverview.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from 'react';
+import { cva } from 'class-variance-authority';
+import { Interweave } from 'interweave';
+// eslint-disable-next-line no-restricted-imports
+import Link from 'next/link';
+import sanitizeHtml from 'sanitize-html';
+
+import { triggerPrototypeToast } from './helpers';
+
+export const ContentOverview = ({ content }) => {
+  const [headings, setHeadings] = useState<string[]>([]);
+
+  useEffect(() => {
+    const parser = new DOMParser();
+    const doc = parser.parseFromString(content, 'text/html');
+    const headingElements = doc.querySelectorAll('h3');
+    const headingTexts = Array.from(headingElements).map(h3 => h3.textContent?.trim() || '');
+    setHeadings(headingTexts);
+  }, [content]);
+
+  const linkClasses = cva('px-2 font-semibold block hover:text-primary text-sm border-l-[3px]', {
+    variants: {
+      active: {
+        true: 'border-primary/80',
+        false: 'border-transparent',
+      },
+    },
+    defaultVariants: {
+      active: false,
+    },
+  });
+
+  return (
+    <div className="space-y-4">
+      <Link href="#" className={linkClasses({ active: true })} onClick={triggerPrototypeToast}>
+        About
+      </Link>
+      {headings.map(heading => {
+        const sanitizedKey = sanitizeHtml(heading, {
+          allowedTags: [], // No tags allowed
+          allowedAttributes: {}, // No attributes allowed
+        }).replace(/[^a-zA-Z0-9-_]/g, '_'); // Replace unsafe characters
+
+        return (
+          <Link
+            href="#"
+            key={sanitizedKey} // Use sanitized and unique key
+            className={linkClasses()}
+            onClick={triggerPrototypeToast}
+          >
+            <Interweave content={heading} />
```

Comment:
Why do we need `Interweave` here, since it's extracted with `textContent` it's not supposed to contain HTML, right?

Reply:
I wasn't sure how safe it was to use `textContent` directly, but looks like it's safe, I've removed Interweave here